### DISCUSSION TITLE
Controlling the degree of vertices

### DISCUSSION THREAD

**Discussion by kenneth-lee-ch** [agrumery/aGrUM#17] [Q&A] [answered]
Is there a way to control the degree of the vertices in the `randomBN` function? I think it is not the same as the `ratio_arc` argument.

**Comment by phwuil** [comment, marked as answer]
I think that you mean the max degree of the vertices.  In aGrUM we have a random generator that constraint the number of parents of a node. The python wrapper has not access to this generator for now.
